flink-runtime/src/test/java/org/apache/flink/runtime/operators/coordination/OperatorCoordinatorSchedulerTest.java

```diff
@@ -466,6 +472,21 @@ public void testBatchGlobalFailoverDoesNotNotifyLocalRestore() throws Exception
         assertThat(coordinator.getRestoredTasks()).isEmpty();
     }
 
+    @Test
+    public void testAllOperatorCoordinatorRegisterMetrics() throws Exception {
+        final DefaultScheduler scheduler =
+                setupTestJobAndScheduler(
+                        new TestingOperatorCoordinator.Provider(testOperatorId), null, null, true);
+        scheduleAllTasksToRunning(scheduler);
+        List<OperatorCoordinatorHolder> operatorCoordinatorHolderList = new ArrayList<>(scheduler.getExecutionGraph()
+                .getJobVertex(testVertexId).getOperatorCoordinators());
+        assertEquals(1, operatorCoordinatorHolderList.size());
```

Comment:
We should use `assertJ` assertions in tests.
See https://flink.apache.org/contributing/code-style-and-quality-common.html#testing.